lib/orientdb_schema_migrator/migration.rb

```diff
@@ -40,33 +40,31 @@ def self.rename_class old_name, new_name
         ODBClient.command "alter class #{old_name} name #{new_name}"
         return true
       else
-        puts "#{class_name} doesn't exist"
         return false
       end
     end
 
-    def self.add_property class_name, property_name, type,property_options={}
+    def self.add_property class_name, property_name, type, property_options={}
+      assert_class_exists!(class_name)
       ODBClient.create_property class_name,property_name,type, property_options
     end
 
     def self.drop_property class_name, property_name
+      assert_class_exists!(class_name)
+      return false unless property_exists?(class_name, property_name)
       ODBClient.command "drop property #{class_name}.#{property_name}"
     end
 
     def self.alter_property class_name, property_name, attribute_name, new_value
+      assert_class_exists!(class_name)
+      return false unless property_exists?(class_name, property_name)
```

Comment:
I think we might want to change all validation check to use methods
assert_class_exists!(class_name)
assert_property_exists!(class_name, property_name)
Is there any reason that we want this method to return false?